test/testdata/fixtures/add-dc/k8ssandra.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: test
 spec:
   cassandra:
-    serverVersion: "4.0.1"
+    serverVersion: "3.11.11"
```

Comment:
Why downgrading from 4.0.1 to 3.11.11 series?

Reply:
3.11.11 provided better stability. There were too many moving parts to work with and this was a simple fix.
We should open a follow up ticket to profile the footprint of C* 4.x vs 3.11 and come up with other adjustments to make 4.0 stable in multi cluster e2e tests.

Reply:
Odd. I've assumed they lowered the resource overhead in 4.x series. Something to investigate later as we really need to test against 4.x

Reply:
agreed 👍

Reply:
it could be due to the fact that CMS isn't available in 4.0 and G1 is the default there. In 3.11, CMS is the default.